src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -498,11 +497,11 @@ func (s *peersSource) flush(
 	}
 	if numSeriesTriedToRemoveWithRemainingBlocks > 0 {
 		iOpts := s.opts.ResultOptions().InstrumentOptions()
-		instrument.EmitAndLogInvariantViolation(iOpts, func(l xlog.Logger) {
-			l.WithFields(
-				xlog.NewField("start", tr.Start.Unix()),
-				xlog.NewField("end", tr.End.Unix()),
-				xlog.NewField("numTimes", numSeriesTriedToRemoveWithRemainingBlocks),
+		instrument.EmitAndLogInvariantViolation(iOpts, func(l *zap.Logger) {
+			l.With(
+				zap.Int64("start", tr.Start.Unix()),
```

Comment:
maybe just `zap.Time("start", tr.Start)` and same with `end`?

Reply:
Don't have enough context for original intent so I'd rather leave the code as it originally was (using time.Unix()).